packages/pwa-kit-runtime/src/ssr/server/express.test.js

```diff
@@ -996,19 +996,46 @@ describe('generateCacheKey', () => {
 })
 
 describe('getRuntime', () => {
+    let originalEnv
+    let originalEval = global.eval
+
+    // Mock the DevSeverFactory via `eval` so we don't have to include it as a dev
+    // dependency which will cause circular dependency warnings.
+    const mockEval = () => ({
+        main: {
+            require: () => ({
+                DevServerFactory: {
+                    name: 'MockServerFactory'
+                }
+            })
+        }
+    })
+
     const cases = [
         {
             env: {},
-            expectedRuntime: DevServerFactory,
+            test: (runtime) => {
+                expect(runtime.name).toEqual('MockServerFactory')
```

Comment:
I'd avoid this `runtime.name` attribute, since it doesn't exist on the mocked Object. Personally, I'd check that the runtime `===` the object that you return from the mock, which is what we're doing in the other test, too.